examples/pom.xml

```diff
@@ -32,7 +32,7 @@ under the License.
 
   <modules>
     <module>beam</module>
-    <module>spark</module>
+<!--    <module>spark</module>-->
```

Comment:
I guess this change is irrelevant to the PR topic.

Reply:
changed! thanks.